Sources/EpoxyLayoutGroups/Types/GroupAnimationStyle.swift

```diff
@@ -0,0 +1,28 @@
+// Created by lindsay_pond on 5/13/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Foundation
+
+
+public typealias GroupAnimationClosure = (_ animations: @escaping () -> Void, _ completion: @escaping (_ completed: Bool) -> Void) -> Void
+public enum GroupAnimationStyle {
+  case animation(LayoutGroupUpdateAnimation)
+  case noAnimation
```

Comment:
We already have a way to specify no animation—specifically you pass `animated: false` to the `setItems(…)` method, so I'm not sure we need this `GroupAnimationStyle` type and we can just have the `LayoutGroupUpdateAnimation`